app/src/helpers/config.ts

```diff
@@ -1,11 +1,16 @@
-export const otcAddress = import.meta.env.VITE_OTC_ADDRESS
-export const RPC: string = import.meta.env.VITE_RPC
-export const WS: string = import.meta.env.VITE_WS
-export const SERVER: string = import.meta.env.VITE_SERVER
-export const TLD: string = import.meta.env.VITE_TLD
-export const prod = import.meta.env.PROD
+import { http, createConfig } from 'wagmi'
+import { harmonyOne, sepolia } from 'wagmi/chains'
+import chain from './chain'
 
 export const debounceTimeout = 500
 
 // mantisa precision to render values in UI
 export const mantisaPrecision = 3
+
+export const config = createConfig({
+  chains: [chain],
+  transports: {
```

Comment:
use anvil (http://127.0.0.1:8545) and harmony (with RPC set from env, default to https://api.harmony.one)

Reply:
Resolved in https://github.com/polymorpher/otc-country/pull/23